designer/server/src/main/scala/pl/touk/nussknacker/ui/process/ScenarioToolbarService.scala

```diff
@@ -54,7 +60,9 @@ final case class ScenarioToolbarSettings(
     topLeft: List[ToolbarPanel],
     bottomLeft: List[ToolbarPanel],
     topRight: List[ToolbarPanel],
-    bottomRight: List[ToolbarPanel]
+    bottomRight: List[ToolbarPanel],
```

Comment:
From the first glance I can't see that we have all combinations of positions covered. Also pattern of naming is counter-intuitive. My suggestion for order and naming:
- topLet
- topCenter
- topRight
- bottomLeft
- bottomCenter
- bottomRight

Reply:
Good point, I've changed this both the config case class and the http api case class and on FE.